src/app/tasks.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from '@angular/core';
+import { Observable, from } from 'rxjs';
+
+export interface ActionTasksElement {
```

Comment:
This type should reflect response from the API, so you shouldn't have `color`, `dueDay` properties here. If there is a need please create a separate type for that.